web/src/components/tasks/search-business-name/AvailableProps.ts

```diff
@@ -0,0 +1,5 @@
+export interface AvailableProps {
```

Comment:
Do we want to start splitting these props out to separate files?

Reply:
We have been leaving the prop types in the same file as the class. 
I was asking if separating them was a pattern we want to start following